doc/source/examples/dem/dem.rst

```diff
@@ -35,33 +36,35 @@ We organize the DEM examples from the simplest to the most complicated example:
 
-      dem_4 [label="Packing in Ball", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/packing-in-ball/packing-in-ball.html"];
+      dem_4 [label="Packing in Ball", href="./packing-in-ball/packing-in-ball.html"];
 
-      dem_5 [label="Small Scale Rotating Drum", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.html"];
+      dem_5 [label="Small Scale Rotating Drum", href="./small-scale-rotating-drum/small-scale-rotating-drum.html"];
     
-      dem_6 [label="Small Scale Rotating Drum Postprocessing", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/small-scale-rotating-drum-postprocessing/small-scale-rotating-drum-postprocessing.html"];
+      dem_6 [label="Small Scale Rotating Drum Postprocessing", href="./small-scale-rotating-drum-postprocessing/small-scale-rotating-drum-postprocessing.html"];
 
-      dem_7 [label="Rotating Drum", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/rotating-drum/rotating-drum.html"];
+      dem_7 [label="Rotating Drum", href="./rotating-drum/rotating-drum.html"];
 
-      dem_8 [label="Rotating Drum with Postprocessing", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/rotating-drum-with-postprocessing/rotating-drum-with-postprocessing.html"];
+      dem_8 [label="Rotating Drum with Postprocessing", href="./rotating-drum-with-postprocessing/rotating-drum-with-postprocessing.html"];
 
-      dem_9 [label="Rotation of Box", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/rotation-of-box/rotation-of-box.html"];
+      dem_9 [label="Rotation of Box", href="./rotation-of-box/rotation-of-box.html"];
 
-      dem_10 [label="Silo", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/silo/silo.html"]; 
+      dem_10 [label="Silo", href="./silo/silo.html"];
 
-      dem_11 [label="Rectangular Hopper", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/rectangular-hopper/rectangular-hopper.html"];
+      dem_11 [label="Rectangular Hopper", href="./rectangular-hopper/rectangular-hopper.html"];
 
-      dem_12 [label="Granular Dam-Break", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/granular-dam-break/granular-dam-break.html"];
+      dem_12 [label="Granular Dam-Break", href="./granular-dam-break/granular-dam-break.html"];
 
-      dem_13 [label="Bunny Drill", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/bunny-drill/bunny-drill.html"];
+      dem_13 [label="Plate Discharge", href="./plate-discharge/plate-discharge.html"];
 
-      dem_14 [label="Granular Mixer", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/granular-mixer/granular-mixer.html"];
+      dem_14 [label="Bunny Drill", href="./bunny-drill/bunny-drill.html"];
+
+      dem_14 [label="Granular Mixer", href="./granular-mixer/granular-mixer.html"];
```

Comment:
I was just skimming through the PR, and this change got me a bit concerned (maybe for nothing). It's just that I think it would work perfectly with the local build (when opening the doc from a local repo), however I think it might not work with the online version and make the link crash. I might be assuming things wrongly, but I think it is worth testing it before merge.

Reply:
No this will not work. This is a discussion we had in the past with @lpsaavedra.
It's impossible to get the image dynamic linking to work with the local website. That's a limitation we cannot get around

Reply:
Perfect, I will remove it then!

Reply:
But why does this work locally?

Reply:
I have no clue to be honest. Like I don't have a rational explanation as to why this works locally but does not work online. I would have to carry out further testing of this. I don't remember what was our conclusion exactly, maybe @lpsaavedra would know.